resources/charts/watcher/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: watcher
+description: A Helm chart for Monitoring Application watcher.
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
I'm guessing this is a version placeholder. Can you use something that is not valid semver so that it can be validated - e.g.:
```yaml
version: 0.0.0
```
or
```yaml
version: version-placeholder
```
so it stands out and hints that it's a placeholder.